SPEC.md

```diff
@@ -242,6 +242,13 @@ distinction in KDL between real numbers, integers, and floating point numbers.
 It's up to individual implementations to determine how to represent KDL
 numbers.
 
+Implementations MUST support number values representable as IEEE754 floating
+point numbers. That is, all implementations must support at least the numbers
+between `-(2^53 - 1)` and `2^53 - 1`. Larger or smaller values MAY error, or
```

Comment:
nit: There are lots of numbers between those two that are not storable in a 64 bit float. Maybe just remove the "That is..." sentence and let implementors refer to IEEE754 directly if needed?

Reply:
ok!